css/layout.css

```diff
@@ -18,7 +22,70 @@
   width: 100%;
 }
 
-@media screen and (min-width: 768px) {
+input {
+  margin: 1rem 0 2rem;
+  height: 2.5rem;
+  border-radius: 0.3rem;
+  border: 0.1rem solid #aeaeaf;
+}
+
+textarea {
+  height: 8rem;
+  margin-top: 1rem;
+  border-radius: 0.3rem;
+  border: 0.1rem solid #aeaeaf;
+}
+
+img {
+  margin-bottom: 1rem;
+  max-width: 100%;
+}
+
+.save-button {
+  width: 100%;
+  justify-content: flex-end;
+}
+
+button {
+  color: white;
+  background-color: #4c2676;
+  margin: 0.75rem 0;
+  border-radius: 0.3rem;
+  border: 0.1rem default #686b6b;
+  padding: 0.5rem;
+}
+
+.header {
+  width: 100%;
+  margin: 0;
+  padding: 2rem 0;
+  color: white;
+  z-index: 0;
+}
+
+.header-background {
+  width: 100vw;
+  padding: 3.15rem 0;
+  background-color: #4c2676;
+  position: absolute;
+  z-index: -1;
+}
```

Comment:
Rather than positioning this on an empty `div`, I would create a `<div>` that surrounds the whole header container, and apply the `background-color` there.